### PR TITLE
Move internal.ContextLogger to be private in ulog

### PR DESCRIPTION
**Pull request by peter-edge**
I think we may actually want some better key here, I think this is just setting the logger at key 0, which isn't very unique. Regardless, this should probably be private inside ulog now.

### CODE REVIEW

File: ulog/log.go [hunk 1]
```diff
@@ -37,6 +36,10 @@ import (
 	"github.com/uber/jaeger-client-go"
 )
 
+const _contextLogger contextKey = iota
```
Comment (yutongp) [inline ulog/log.go]: I thought context key better to be unique. If anyone use 0 or used 0, value will be overwrote.

Comment (peter-edge) [inline ulog/log.go]: Fixed

**Comment by sectioneight** [comment]
So, ulog is gone now, do we close this? @akshayjshah

**Comment by akshayjshah** [comment]
Yep, I think so.

**Pull request closed without merging**